src/debugger/launch.ts

```diff
@@ -254,12 +252,24 @@ export function createDarwinTestConfiguration(
     };
 }
 
+function launchConfigKeysToUpdate(): string[] {
+    const keysToUpdate = ["program", "cwd", "preLaunchTask"];
```

Comment:
These keys are updated in addition to `env` when runtime path is updated, and the dialog doesn't reflect this.

Reply:
These should only get updated if there is a new version of the extension. If someone changed any of these they would break the launch config. I doubt these will change from this point on, but they have changed between earlier versions of the extension.

Reply:
I mean the message here should also be adjusted (& simplified)👀

https://github.com/swift-server/vscode-swift/blob/2a33ee8c2ac46072d925abf618321cc3f8c4fc38/src/WorkspaceContext.ts#L97

Reply:
Given the advance nature of this feature I think the message is ok

Reply:
Can we extract "runtimePathEnvKey" to be an exported function? It seems to be duplicated in multiple functions.

Reply:
makes sense

Reply:
> Can we extract "runtimePathEnvKey" to be an exported function? It seems to be duplicated in multiple files.

done